packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -277,27 +277,39 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
         isSignUpComplete: res["isSignUpComplete"],
         nextStep: AuthNextSignUpStep(
             signUpStep: res["nextStep"]["signUpStep"],
-            codeDeliveryDetails: res["nextStep"]["codeDeliveryDetails"],
+            codeDeliveryDetails: res["nextStep"]["codeDeliveryDetails"] != null
+            ? AuthCodeDeliveryDetails(
+              attributeName: res["nextStep"]["codeDeliveryDetails"]["attributeName"] ?? null,
```

Comment:
Isn't
```dart
attributeName: res["nextStep"]["codeDeliveryDetails"]["attributeName"] ?? null
```
the same as saying if it's `null` make it `null` thus functionally equivalent to just
```dart
attributeName: res["nextStep"]["codeDeliveryDetails"]["attributeName"]
```